File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_rtnanarn.py
"""Reference emit for RTNANARN — CLBS RTNANA00-shaped status-weight sum → 23.75."""
# EXPECTED: 23.75
WEIGHTS = [1.25, 1.25, 1.25, 2.50, 2.50, 5.00, 10.00]
print(f"{sum(WEIGHTS):.2f}")

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_rtnanarn.cs
// EXPECTED: 23.75
// Reference emit for RTNANARN — CLBS return-code analysis sequential weight sum.
using System;
public static class RtnanarnRef {
  public static void Main() {
    double[] weights = {1.25, 1.25, 1.25, 2.50, 2.50, 5.00, 10.00};
    double total = 0.0;
    foreach (var v in weights) total += v;
    Console.WriteLine(total.ToString("0.00"));
  }
}
